### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by fpsapc**
#### Here are the thing that I made in working in this project.

- Splash screen

   - A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.

- Sign up and log in pages

   - The user should be able to register in the app with full name, email and password (all mandatory).
   - The user can log into the app using email and password.
   - If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).
Home page (categories page)

- When the user logs in, they are presented with the categories page.
    - For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
    - When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
    - There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

- Transactions page

    - For a given category, the list of transactions is presented, ordered by the most recent.
    - At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
    - There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
    - When the user clicks on the "Back" button (<), the user navigates to the home page.

- "Add a new category" page

   - The user fills out a form to create a new category, indicating their name and icon (both mandatory).
   - The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
   - When the user clicks on the "Back" button (<), the user navigates to the home page.

- "Add a new transaction" page

    - The user fills out a form to create a new transaction with:
    - name (mandatory)
    - amount (mandatory)
    - categories (mandatory at least one)
    - The user clicks (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
    - When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.

- Testing requirements
     - Create unit and integration tests for all the most important components of your RoR application.
  
- Also deploy the app using render and add a link to the video presentation as well in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,218 @@
...
+- **Add a new category page**
+- **Add a new transaction page**
+- **Use devise for user authentication**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- <a href="https://budget-app-g33j.onrender.com/">Live Demo</a>
+- [Video Presentation](https://www.loom.com/share/930e60a53bb9417daf0b622c47d1d078?sid=4a91d503-5f1c-4751-adbe-92a2dd040542)
+
```
Comment (zieeco) [inline README.md:85]: - Hey, your video is missing a highlight about a piece of code that you find interesting or proud of. Please, record another video to implement these missing features.

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+  padding: 0;
+}
+
+body {
+  background-color: rgb(241, 241, 241);
+}
+
+nav {
+  /* display: grid;
+  grid-template-columns: repeat(3, 1fr); */
```
Comment (zieeco) [inline app/assets/stylesheets/application.css]: - Please, let's remove this commented code to make our codebase clean and maintainable.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @fpsapc,



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice with the budget app ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions

- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (fpsapc) [reply README.md:85]: @zieeco : hello dear, I already highlights that piece of code in last part of the video, that is a method written in application_controllers.rb to override the devise's after_sign_in_path_for method. Are you not agreed with that please tell me so I can move forward.

File: app/assets/stylesheets/application.css (hunk 2 above)
Comment (fpsapc) [reply app/assets/stylesheets/application.css]: @zieeco already remove the commented code. Thanks for highlight.

File: README.md (hunk 1 above)
Comment (zieeco) [reply README.md:85]: - Hey, sorry for the oversight from my end.

Comment (fpsapc) [reply README.md:85]: Don't worry it's ok. I am going to submit again.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @fpsapc, 




Good job so far 👍 Your project is complete! There's nothing else to say other than,  it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 
✅ Nice implementing the changes! 👏 

## Optional suggestions

- N/A


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**